### DISCUSSION TITLE
ElasticSearch Datasource doesn't provide data when creating alerts

### DISCUSSION THREAD

**Discussion by tleuschner** [grafana/grafana#59692] [Alerting]
I configured a ElasticSearch datasource which seems to be working properly in the dashboards panel and the explore tab. However when trying to create an alert with the ElasticSearch datasource, there seems to be a problem.
First I get the following error, when initially selecting the datasource
```
Failed to evaluate queries and expressions: [plugin.downstreamError] failed to query data: type assertion to string failed
```
After that, when changing query, the error seems to disappear but no data is returned.

Do you have any idea on how to fix this issue?

Setup:
grafana Version: 9.2.5
Elastic Search Datasource
- SigV4 Auth using the AWS SDK Default
- ElasticSearch version 7.10+

Kind regards

**Comment by gabor** [comment]
i created a github-issue for this, that will be a better place to discuss this problem, the issue is at: https://github.com/grafana/grafana/issues/61848

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
